app/statistics/server/lib/statistics.ts

```diff
@@ -108,11 +243,11 @@ export const statistics = {
 		statistics.totalThreads = Messages.countThreads();
 
 		// Teams statistics
-		statsPms.push(
-			Team.getStatistics().then((teamStats) => {
-				statistics.teams = teamStats;
-			}),
-		);
+		// statsPms.push(
```

Comment:
Remove commented code if not in use

(I actually cannot see this at other line 👀 is inteded to be commented?)

Reply:
Yeah, the function didn't exist on the type or could i find anywhere where it was defined, so i removed it.